components/Button/Button.tsx

```diff
@@ -9,7 +9,7 @@ export interface ButtonProps {
   onClick?(): any;
 }
 
-const rootClassName = 'fui-button';
```

Comment:
I'm glad you're doing this now!

We can't use a root class name and string concat's with tailwind classes. These will get purged! 
(they may be excluded from the purge now, but we should really get back to purging everything that's unused)

To be honest it might be worth moving these styles out of the css file and just put them directly into the component as pure tailwind classes. I think we decided in a previous jam that we were going to start moving away from css files.